src/clone.d

```diff
@@ -199,6 +199,21 @@ extern (C++) FuncDeclaration buildOpAssign(StructDeclaration sd, Scope* sc)
     StorageClass stc = STCsafe | STCnothrow | STCpure | STCnogc;
     Loc declLoc = sd.loc;
     Loc loc = Loc(); // internal code should have no loc to prevent coverage
+
+    // One of our sub-field might have `@disable opAssign` so we need to
+    // check for it.
+    for (size_t i = 0; i < sd.fields.dim; i++)
```

Comment:
I'm merely moving things around and adding documentation here.
I have further plans for this part of the code so I'll keep this comment in mind for later.